packages/base-map/src/index.tsx

```diff
@@ -22,24 +22,34 @@ import MarkerWithPopup from "./MarkerWithPopup";
  * with an id are added to the control.
  */
 type Props = React.ComponentPropsWithoutRef<React.ElementType> & {
+  /** A URL pointing to the vector tile specification which should be used as the main map.  */
   baseLayer?: string;
+  /** A [lat, lon] position to center the map at. */
   center?: [number, number];
+  /** Whether or not to override the default MapLibre 100% height to be 90vh */
   forceMaxHeight?: boolean;
```

Comment:
I REALLY don't like this prop and it should definitely be removed, but I know what is happening.

In storybook, previously, a style tag was inserted that set the leaflet container height to 480px. 
I think we still want to set some height, so that the map shows in storybook. As written, the map is set to take 100% of the height, but no height is defined in the parent tags so basically the height is zero. This is not the case and won't be an issue in OTP-RR where the height is defined by the successive map containers.

What we should do here is to use the `className` and `style` props. In storybook, define a `styled(BaseMap)` with the `height` of 480px, 90vh or whatever and render the styled component in each story.